challenge-19/submissions/evassilyev/solution-template.go

```
@@ -0,0 +1,84 @@
+	}
+
+	return result
+}
+
+// ReverseSlice returns a new slice with elements in reverse order.
+func ReverseSlice(slice []int) []int {
+	lenth := len(slice)
+	if lenth == 0 {
+		return slice
+	}
+	reversed := make([]int, lenth, lenth)
```

golangci-lint (2.13.2)

[error] S1019: should use make([]int, lenth) instead

(staticcheck)